### PR TITLE
[QEff Finetune] Correction in data type of loss

### PR DESCRIPTION
**Pull request by quic-swatia**
In case of DDP, if size of dataset (train dataset or validation  dataset) is smaller than the degree of DDP, then the loss value for padded samples was coming out to be in float. It's handled with this change.

### CODE REVIEW

**Review by quic-meetkuma — COMMENTED** [review]
LGTM, Make a JIRA and reference the gerrit on the JIRA. Post that we are good to go.

**Review by quic-akuruvil — APPROVED** [review]
LGTM

**Review by quic-meetkuma — APPROVED** [review]
Thanks for spotting the issue and fixing it. :)
